jans-auth-server/model/src/main/java/io/jans/as/model/crypto/Certificate.java

```diff
@@ -90,13 +142,17 @@ public JSONArray toJSONArray() throws JSONException {
     public String toString() {
         try {
             StringWriter stringWriter = new StringWriter();
-            try (JcaPEMWriter pemWriter = new JcaPEMWriter(stringWriter)) {
```

Comment:
try-resource statement looks better, is there any reason with replacing it with manual `close()`?

Reply:
It was made according to Sonar requirements. As this code has been implemented in other branches and other PRs, it has been updated during removing Sonar warns.
But it could be back. of course.